lib/Cake/View/Helper/FormHelper.php

```diff
@@ -379,6 +379,13 @@ public function create($model = null, $options = array()) {
 		$this->inputDefaults($options['inputDefaults']);
 		unset($options['inputDefaults']);
 
+		if (isset($options['action'])) {
+			trigger_error('Using key `action` is deprecated, use `url` directly instead.', E_USER_DEPRECATED);
+		}
+		if (isset($options['url']) && isset($options['url']['action'])) {
```

Comment:
@dereuromark Please fix it. Don't mind me :)
I must sleep. It's 2:00 am here (JST).